src/main/java/seedu/address/logic/parser/CsvAttendanceParser.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.logic.parser;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import seedu.address.logic.parser.exceptions.ParseException;
+/**
+ * Parses attendance data from a CSV field, extracting the weeks present as a set of integers.
+ */
+public class CsvAttendanceParser {
+    /**
+     * Parses the weeksPresent field into a set of integers.
+     *
+     * @param weeksField string representing weeks present, separated by commas
+     * @return a set of integers representing weeks present
+     * @throws ParseException if any week value is invalid
+     */
+    public static Set<Integer> parseWeeksPresent(String weeksField) throws ParseException {
+        Set<Integer> weeksPresent = new HashSet<>();
```

Comment:
I feel that we can include a `requireNonNull` check here just in case. The same applies to the other parsers?